### DISCUSSION TITLE
/health returns 404 in versions >= 0.17.0

### DISCUSSION THREAD

**Discussion by jadamon42** [tchiotludo/akhq#1222] [Q&A] [answered]
I assume I must be doing something wrong, but I'm not sure what it could be. I've been trying to use AKHQ for my kafka cluster, but have trouble with any version >= 0.17.0

I was trying to do this in AWS through ECS and fargate and kept getting this same issue with the health check, so decided to try to figure it out locally first. Same thing. If I do:

```docker run -p 8080:8080 -v /path/to/some/config.yml:/app/application.yml tchiotludo/akhq:0.16.0```

and then: 

```curl http://localhost:8080/health``` 

I get a 200, as expected. As soon as I upgrade to 0.17.0 (or any version thereafter) and do that same call, I get a 404. Here's my yml file:

```
akhq:
  connections:
    msk-cluster:
      properties:
        bootstrap.servers: "kafka:9098"
        security.protocol: SASL_SSL
        sasl.mechanism: AWS_MSK_IAM
        sasl.jaas.config: software.amazon.msk.auth.iam.IAMLoginModule required awsDebugCreds=true;
        sasl.client.callback.handler.class: software.amazon.msk.auth.iam.IAMClientCallbackHandler
```

My most recent cluster uses IAM for authentication, but I experienced the same issue over plaintext. Doesn't seem that the actual connection to the kafka cluster matters for the health check anyway.

I didn't want to open a bug, because I suspect this is an issue on my end. I just can't seem to figure it out. Do I need to add something to the config file in order to enable the `/health` endpoint? Any help would be appreciated, thanks.

**Comment by tchiotludo** [comment, marked as answer]
The port have changed between this version, now it's `curl http://localhost:28080/health`

**Reply by jadamon42** [reply]
I tried updating my port, and got the following: 

```
> curl http://localhost:28080/health
curl: (56) Recv failure: Connection reset by peer
```

I even tried forwarding port 28080 when running my docker image. Also, I tried changing the health check port for the target in my ECS cluster service and no luck there either.

I am noticing this in the Dockerfile in this repo:

```
HEALTHCHECK --interval=1m --timeout=30s --retries=3 \
  CMD curl --fail http://localhost:8080/health || exit 1
```

The Dockerfile is still using port 8080. Maybe it's some other port? I saw some mention of port `28081` in the documentation but that didn't work for me either. Let me know if you have any idea what this might be

**Reply by tchiotludo** [reply]
oups, the good one : 
`curl http://localhost:28081/health`

![image](https://user-images.githubusercontent.com/2064609/193670277-232dcbd8-9095-4109-9413-b2ddc3b9c7a7.png)

**Comment by justinmchase** [comment]
I'm getting this error for port 28081 also, can we get a full example of the correct probe settings?

**Reply by tchiotludo** [reply]
humm strange ... the example I give is the latest version that is working on my side. 
Maybe you can add some details ?

**Reply by justinmchase** [reply]
Sorry for the noise, unbeknownst to me we had configured the a prefix for the path so it was `/akhq/health` for me instead of just `/health`.
